src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -240,6 +240,23 @@ private BoundAttribute BindAttributeCore(AttributeSyntax node, NamedTypeSymbol a
                     {
                         Error(diagnostics, ErrorCode.ERR_AttributeCtorInParameter, node, attributeConstructor.ToDisplayString(SymbolDisplayFormat.CSharpErrorMessageFormat));
                     }
+
+                    for (var i = 0; i < boundConstructorArguments.Length; i++)
+                    {
+                        var parameter = GetCorrespondingParameter(i, attributeConstructor.Parameters, argsToParamsOpt, expanded);
```

Comment:
>GetCorrespondingParameter(i, attributeConstructor.Parameters, argsToParamsOpt, expanded) [](http://example.com/codeflow?start=40&length=88)

Is this call still needed?

Reply:
Good catch, it wasn't. But this whole code went away.